### PR TITLE
Tup #59 feat error messages for catch errors

### PR DESCRIPTION
**Pull request by rels742**

### CODE REVIEW

File: src/components/posts/CommentItem.js [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
 const CommentItem = ({ comment, setUser }) => {
   const navigate = useNavigate();
-  
+
   const handleClick = e => {
     client
       .get(`/user/${comment.userId}`)
       .then(res => setUser(res.data.data.user))
-      .catch(err => console.log(err));
+      .catch(err => console.log(err.response));
```
Comment (Auenc) [inline src/components/posts/CommentItem.js:12]: Let's change this to a console.error instead

File: src/components/posts/PostItem.js [hunk 2]
```diff
@@ -103,21 +110,21 @@ const PostItem = ({ post, userId, setPostResponse, setPost, setUser, user }) =>
...
     }
-  }
+  };
 
   const handleClick = (e, id = post.userId) => {
     client
       .get(`/user/${id}`)
       .then(res => setUser(res.data.data.user))
-      .catch(err => console.error);
+      .catch(err => console.log(err.response));
```
Comment (Auenc) [inline src/components/posts/PostItem.js:127]: Let's use console.error instead here

File: src/components/profile/Profile.js [hunk 3]
```diff
@@ -1,79 +1,89 @@
...
+      return;
     }
 
-    const handleChange = (event) => {
-        event.preventDefault()
-        const { value, name } = event.target
+    client
+      .patch('/user/myprofile', reqBody)
+      .then(res => setUser(res.data.data.user))
+      .catch(err => console.log(err.response));
```
Comment (Auenc) [inline src/components/profile/Profile.js:39]: Let's change this to a console.error

**Review by Auenc — CHANGES_REQUESTED** [review]
Some minor changes

**Review by Auenc — APPROVED** [review]

**Review by CoderManSam — APPROVED** [review]
LGTM

**Pull request merged**